libs/ui/src/ui_strings/toggle_audio_button.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Button } from '../button';
+import { P } from '../typography';
+import { useAudioControls } from '../hooks/use_audio_controls';
+import { useAudioContext } from './audio_context';
+import { Icons } from '../icons';
+import { appStrings } from './app_strings';
+import { WithAltAudio } from './with_alt_audio';
+
+export function ToggleAudioButton(): React.ReactNode {
+  const isAudioEnabled = useAudioContext()?.isEnabled || false;
+  const audioControls = useAudioControls();
+
+  const statusText = isAudioEnabled ? (
+    <React.Fragment>
+      <Icons.SoundOn /> {appStrings.noteVoterSettingsAudioUnmuted()}
+    </React.Fragment>
+  ) : (
+    <React.Fragment>
+      <Icons.SoundOff /> {appStrings.noteVoterSettingsAudioMuted()}
+    </React.Fragment>
+  );
+
+  return (
+    <div>
```

Comment:
Curious if there's an accessibility benefit to using `<div>` over `<React.Fragment>` here?

Reply:
no benefit there - just wanted to make sure any layout styling applied at the parent level doesn't affect this component's internal layout (e.g. the flex styling in the `<Buttons>` container in `AudioSettings`)